app/src/main/java/ch/hikemate/app/ui/map/MapScreen.kt

```diff
@@ -551,7 +578,9 @@ fun MapSearchButton(onClick: () -> Unit, modifier: Modifier = Modifier, enabled:
           ),
       enabled = enabled) {
         Text(
-            text = LocalContext.current.getString(R.string.map_screen_search_button_text),
+            text = if (shortText) "Search here" else "Search for hikes",
+            // text = "Search
+            // here",//LocalContext.current.getString(R.string.map_screen_search_button_text),
```

Comment:
Warning: I think you forgot to actually use the `strings.xml` here

Reply:
Öhh yes, i forgot to push that. Changed.